docs/better-burgers/index.md

```diff
@@ -0,0 +1,357 @@
+# Better Burgers
+
+Cafe Emoji is still the hottest restaurant in town, despite the fact that all
+burgers are exactly the same. Madame Jellobutter isn't one to rest on her
+laurels, though, so she decides to kick it up a notch by allowing customers to
+choose the toppings on their burgers.
+
+Currently, your `Item.t` variant type looks something like this:
+
+<<< Item.re#starting-item-t
+
+## `burger` record type
+
+Add a new `burger` *record* type and then add that new type as an argument for
+the `Burger` constructor of `Item.t`:
+
+<<< Item.re#add-burger-type{1-5,9}
+
+The fields in the `burger` record are:
+
+| Name | Type | Meaning |
+| ---- | ---- | ------- |
+| lettuce | bool | if `true`, include lettuce |
```

Comment:
nit: maybe this should be either
- `bool`
or
- boolean
?

Same with int: either `int` or integer. 🤔

Reply:
I opted to just put the field names and types in backticks in https://github.com/melange-re/melange-for-react-devs/pull/21/commits/ad5f95923b2b8cdbc117704b3b8b4434bdb92662